lib/transbank/sdk/webpay/webpay_plus/mall/mall_transaction.rb

```diff
@@ -0,0 +1,89 @@
+            return ::Transbank::Webpay::WebpayPlus::TransactionCreateResponse.new(json) if resp.kind_of? Net::HTTPSuccess
+            raise Errors::TransactionCreateError.new(json['error_message'], resp.code)
+          end
+
+          def commit(token:, options:nil)
+            api_key = options&.api_key || default_integration_params[:api_key]
+            commerce_code = options&.commerce_code || default_integration_params[:commerce_code]
+            integration_type = options&.integration_type || default_integration_params[:integration_type]
+            base_url = integration_type.nil? ? WebpayPlus::Base::integration_type[:TEST] : WebpayPlus::Base.integration_type_url(integration_type)
+            url = base_url + "#{COMMIT_TRANSACTION_ENDPOINT}/#{token}"
+            headers = webpay_headers(commerce_code: commerce_code, api_key: api_key)
+            resp = http_put(uri_string: url, body: {}, headers: headers)
+            json = JSON.parse(resp.body)
+            return ::Transbank::Webpay::WebpayPlus::MallTransactionCommitResponse.new(json) if resp.kind_of? Net::HTTPSuccess
+            raise Errors::TransactionCommitError.new(json['error_message'], resp.code)
+          end
+
+
+          def refund(token:, buy_order:, child_commerce_code:, amount:, options:nil)
+            api_key = options&.api_key || default_integration_params[:api_key]
+            commerce_code = options&.commerce_code || default_integration_params[:commerce_code]
+            integration_type = options&.integration_type || default_integration_params[:integration_type]
+            base_url = integration_type.nil? ? WebpayPlus::Base::integration_type[:TEST] : WebpayPlus::Base.integration_type_url(integration_type)
+            url = base_url + "#{REFUND_TRANSACTION_ENDPOINT}".gsub(':token', token)
+            headers = webpay_headers(commerce_code: commerce_code, api_key: api_key)
+            body = {
+              buy_order: buy_order,
+              commerce_code: child_commerce_code,
+              amount: amount
+            }
+            resp = http_post(uri_string: url, body: body, headers: headers, camel_case_keys: false)
+            json = JSON.parse(resp.body)
+            return ::Transbank::Webpay::WebpayPlus::TransactionRefundResponse.new(json) if resp.kind_of? Net::HTTPSuccess
+            raise Errors::TransactionRefundError.new(json['error_message'], resp.code)
+          end
+
+          def status(token:, options: nil)
+            api_key = options&.api_key || default_integration_params[:api_key]
+            commerce_code = options&.commerce_code || default_integration_params[:commerce_code]
+            integration_type = options&.integration_type || default_integration_params[:integration_type]
+            base_url = integration_type.nil? ? WebpayPlus::Base::integration_type[:TEST] : WebpayPlus::Base.integration_type_url(integration_type)
+            url = base_url + "#{TRANSACTION_STATUS_ENDPOINT}".gsub(':token', token)
```

Comment:
It doesn't make a lot of sense to interpolate a string that only contains the interpolated value.  `base_url + "TRANSACTION_STATUS_ENDPOINT.gsub(':token', token)` should work just as fine

Reply:
oh, good catch, that must be a leftover from some previous iteration. Will fix